ieee_tec_bot/ieee_tec_bot.py

```diff
@@ -275,10 +275,11 @@ def informationHandler(bot, update):
     elif userState[update.message.chat_id][0] == guideScreen:
         if update.message.text in customKeyboards[guideScreen][0]:
             #If Membership info selected, get the info from the info module
-            #replyText=info.membershipSteps() 
-            replyText = "Mostrando pasos para pertenecer a IEEE"
-            goToScreen(bot, update, message=replyText)#if a photo or document needs to be added , just add the parameter photo=the photo or document= the document
-
+            replyText=info.membershipSteps() 
+            membershipDocument= open("MembresíaIEEE.pdf","rb")
```

Comment:
This won't work since the file is not in the same folder.
Try with:
```
membershipDocument= open("../docs/MembresiaIEEE.pdf","rb")
```
But since the file name can be modified I suggest creating a variable in the "config.py" file with the relative path. So the call will be more like:
```
membershipDocument= open(config.membershipPath,"rb")
```